src/databricks/labs/ucx/config.py

```diff
@@ -58,6 +58,9 @@ class WorkspaceConfig:  # pylint: disable=too-many-instance-attributes
     # List of workspace ids ucx is installed on, only applied to account-level installation
     installed_workspace_ids: list[int] | None = None
 
+    # Threshold for row count comparison during data reconciliation, in percentage
+    reconciliation_threshold: int = 5
```

Comment:
```suggestion
    reconciliation_failure_tolerance_percent: int = 5
```

out API guidelines dictate that unit name has to be part of the field name. if it's `_percent`, then probably we need to rename the field a bit as well. just has to sound natural. suggested what _might_ sound natural, pick any other alternative with `_percent` suffix